### PR TITLE
feat: Implement instance traps

### PR DESCRIPTION
**Pull request by jubianchi**

### CODE REVIEW

**Comment by jubianchi** [comment]
bors try

File: wasmer/instance.go [hunk 1]
```diff
@@ -41,13 +41,12 @@ func NewInstance(module *Module, imports *ImportObject) (*Instance, error) {
 	runtime.KeepAlive(module.store)
 	runtime.KeepAlive(imports)
 
-	if instance == nil {
-		return nil, newErrorFromWasmer()
+	if traps != nil {
```
Comment (Hywan) [inline wasmer/instance.go:44]: :+1:

File: wasmer/instance_test.go [hunk 2]
```diff
@@ -82,13 +82,27 @@ func TestInstanceMissingImports(t *testing.T) {
...
+}
+
+func TestInstanceTraps(t *testing.T) {
+	engine := NewEngine()
+	store := NewStore(engine)
+	module, err := NewModule(
+		store,
+		[]byte(`
+			(module
+              (start $start_f)
```
Comment (Hywan) [inline wasmer/instance_test.go:97]: Indentation must be fixed :-).

**Review by Hywan — APPROVED** [review]
Apart from the comment, it looks good to me!

**Pull request merged**